packages/actor-init-sparql/config/config-default.json

```diff
@@ -87,6 +94,11 @@
             "@type": "MediatorCombineUnion",
             "cc:Mediator/bus": { "@id": "cbss:Bus/SparqlSerialize" },
             "cmcu:Mediator/CombineUnion/field": "mediaTypes"
+          },
+          "mediatorContextPreprocess": {
+            "@id": "ex:mediatorContextPreprocess",
+            "@type": "MediatorCombinePipeline",
```

Comment:
Why is this a pipeline mediator? Doesn't each of the preprocess actors simply handle a single source and only returns true on the test if it's the applicable actor? This actor would actually make more sense to me if the thing they have in their `test` function now would be in their `run` function. What am I missing?

Reply:
You're probably confusing things. Here's an ASCII figure explaining the structure:

```
Actor: SPARQL init
  |
  \ Mediator (pipeline): Context Preprocess
    | Actor: Context Preprocess - RDF Source Identifier
    \ Mediator (race): RDF Source Identifier
      | Actor: RDF Source Identifier: File
      | Actor: RDF Source Identifier: SPARQL
      | Actor: RDF Source Identifier: Hypermedia
```